src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyReference.cs

```diff
@@ -10,12 +10,12 @@ public KeyReference() : base()
             ReferenceType = "KeyReference";
         }
 
-        public KeyReference(string uri) : base(uri)
+        public KeyReference([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("Uri")] string uri) : base(uri)
```

Comment:
In the `src` source, it should use `using` with constants from StringSyntaxAttribute. This form with fully qualified name and inlined constant is for `ref` source.